### PR TITLE
Added check for CT alive with a Defuse Kit, when defusing without one

### PR DESCRIPTION
**Pull request by enghausen**

### CODE REVIEW

File: scripting/retakes_instadefuse.sp [hunk 1]
```diff
@@ -24,7 +24,7 @@ public Plugin myinfo =
     name = "[Retakes] Instant Defuse",
     author = "B3none",
     description = "Allows a CT to instantly defuse the bomb when all Ts are dead and nothing can prevent the defusal.",
-    version = "1.4.0",
+    version = "1.4.1",
```
Comment (b3none) [inline scripting/retakes_instadefuse.sp:27]: This should be version `1.5.0` as it's a feature addition, not a hotfix.

File: scripting/retakes_instadefuse.sp [hunk 2]
```diff
@@ -118,6 +118,24 @@ void AttemptInstantDefuse(int client, int exemptNade = 0)
...
+	{
+		for (int i = 0; i <= MaxClients; i++)
+		{
+			if (IsValidClient(i) && HasDefuseKit(i))
+			{
+				for (int j = 0; j <= MaxClients; j++)
+				{
+					if (IsValidClient(j))
+					{
+						PrintToChat(j, "%T", "SomeoneAliveHaveKit", j, MESSAGE_PREFIX);
```
Comment (b3none) [inline scripting/retakes_instadefuse.sp:131]: can we switch `"SomeoneAliveHaveKit"` to `"TeammateHasDefuseKit"`

File: translations/instadefuse.phrases.txt [hunk 3]
```diff
@@ -23,4 +23,10 @@
         "#format"    "{1:s},{2:.1f}"
         "en"        "{1} There were {2} seconds left of the bomb. Counter Terrorists win."
     }
+	
+    "SomeoneAliveHaveKit"
+    {
+        "#format"    "{1:s}"
+        "en"        "{1} There's no time to defuse without a Defuse Kit!"
```
Comment (b3none) [inline translations/instadefuse.phrases.txt:30]: `"{1} There's no time to defuse without a Defuse Kit!"` to `"{1} You have no time! Your teammate has a Defuse Kit!"`

**Comment by b3none** [comment]
Can you also remove the `.smx` from `plugins/`

**Comment by enghausen** [comment]
Thanks for the great Code Review! I just did some last testing yesterday (Retake, Pistol Only, 9 Players) and everything seems to work as intended!

Feel free to leave more comments if you have something!

**Comment by b3none** [comment]
That's merged, thank you for your contribution!

**Comment by TandelK** [comment]
@b3none I think it will be better if we remove that player has defuse kit because it will make CT person more easily know they still have a chance but instead let the round get completed. This will bring more awareness otherwise this easy saying that Defuse Kit is available will not make good impact for players watching their loadout and game sense properly. This are my views

**Pull request merged**